docs/service-catalog/docs/014-details-etcd-database.md

```diff
@@ -0,0 +1,110 @@
+
+| Property name              | Description |
+|---------------------------------------------------|---|
+| **etcd-operator.backupOperator.enabled**            | If set to true, the [etcd-operator][etcd-operator-chart] chart installs the [etcd-backup-operator][etcd-backup-operator-chart]. The etcd-operator also creates the [Secret][abs-creds] with the **storage-account** and **storage-key** keys.  |
+| **etcd-operator.backupOperator.abs.storageAccount** | The name of the storage account for the Azure Blob Storage (ABS). It stores the value for the **storage-account** Secret key. |
+| **etcd-operator.backupOperator.abs.storageKey**     | The key value of the storage account for the ABS. It stores the value for the **storage-key** Secret key. |
+| **global.etcdBackupABS.containerName**              | The ABS container to store the backup. If set, the Service Catalog [sub-chart][sc-backup-sub-chart] installs the CronJob which executes periodically the [Etcd Backup][etcd-backup-app] application. For more information on how to configure the backup CronJob, see the [Etcd Backup][etcd-backup-app-readme] documentation. |
+
+> **NOTE:** If you to set the **storageAccount**, **storageKey**, and **containerName** properties, the **etcd-operator.backupOperator.enabled** must be set to `true`. 
+
+### Restore
+
+Follow this instruction to restore an `etcd` cluster from the existing backup.
+
+> **NOTE:** You must have the backup files created by the CronJob backup application from the previous section.
+
+1. Install the etcd-restore-operator:
+```bash
+kubectl create -f assets/etcd-restore/operator-deploy.yaml
+```
+
+2. Create the EtcdCluster Custom Resource Definition:
+```bash
+kubectl create -fassets/etcd-restore/restore-crd.yaml 
+```
+
+3. Export the **CONFIG_MAP_NAME** environment variable. Get the ConfigMap name from [here][sc-backup-job-cfg-name]:
+```bash
+export CONFIG_MAP_NAME={CONFIG_MAP_NAME}
+```
+
+4. Export the **ABS_PATH** environment variable with the path to the last successful backup file.
+```bash
+export ABS_PATH=$(kubectl get cm -n kyma-system $CONFIG_MAP_NAME -o=jsonpath='{.data.abs-backup-file-path-from-last-success}')
+```
+
+5. Export the **SECRET_NAME** environment variable with the Secret name to the ABS. Get the Secret name from [here](../../../resources/core/values.yaml#L34):
+```bash
+export SECRET_NAME={SECRET_NAME_FROM_VALUES_YAML}
+```
+
+6. Create the EtcdRestore Custom Resource which triggers a restore process:
+```bash
+sed -e "s|<full-abs-path>|$ABS_PATH|g" \
+    -e "s|<abs-secret>|$SECRET_NAME|g" \
+    assets/etcd-restore/restore-cr.yaml \
+    | kubectl create -f -
+```
+
+Now the etcd-restore-operator restores a new cluster from the backup
+
+7. See the status of the Pods and wait until all of them are ready. Check the [number][sc-etcd-cluster-no] of the Pods which should be in `RUNNING` state before going to the next step:
```

Comment:
Maybe, we can add information what is currently going on, for example "Now, restore operator is going to kill currently running etcd pods and create anew ones."